code/main.cpp

```diff
@@ -1,12 +1,83 @@
+/*
+ * This main is used to test the canbus or beetle class.
+ */
+
 #include <hwlib.hpp>
-#include <qik_2s12v10.hpp>
+#include <beetle.hpp>
+#include <comm.hpp>
 
 int main(void) {
     // kill the watchdog
     WDT->WDT_MR = WDT_MR_WDDIS;
     hwlib::wait_ms(1000);
 
-    while (true) {
+    // canbus beetle testing
+    bool beetle_canbus = false;
+    // beetle testing
+    bool test_set_speed = true;
+    //bool test_get_configuration_parameter = true;
```

Comment:
This code does not need to be commented 
```suggestion
    bool test_get_configuration_parameter = true;
```

Reply:
It will give an error for unused variable. It will be needed later

Reply:
in new commit it is fixed